go.mod

```diff
@@ -41,7 +41,6 @@ replace (
 	github.com/hashicorp/vault/sdk => github.com/hashicorp/vault/sdk v0.1.14-0.20201116234512-b4d4137dfe8b
 	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3
 	gomodules.xyz/jsonpatch/v2 => github.com/gomodules/jsonpatch/v2 v2.2.0
-	google.golang.org/grpc => google.golang.org/grpc v1.35.0
```

Comment:
@nixpanic just wondering why we were sticking to `1.35` even while referring `1.38` above.. I am not sure the exact reason for sticking to that version.

Reply:
I don't know either, it seems it was introduced with commit 5aca04d :wink: 

Hopefully we can drop many `replace` lines and just use the latest versions of dependencies.